list_of_prime_numbers/main.py

```diff
@@ -1,10 +1,18 @@
 
 def _is_prime(number):
-    pass
+    for i in range(2, number):
+        if (number % i) == 0:
+            return False
+    return True
 
 
 def list_of_prime_numbers(max_number):
-    pass
+    result = []
+    a = max_number + 1
+    for i in range(2, a):
```

Comment:
you can actually try to make this shorter by writing it like this:
`for i in range(2, max_number + 1)`
and taking out line 11 :D that way, you don't need to create a variable.